packages/controller/main.js

```diff
@@ -5352,7 +5352,8 @@ function init(compactGroupId) {
                 `${hostLogPrefix} Your logging path "${e.path}" was invalid, it has been changed to "${fixedLogPath}"`
             );
         } else {
-            console.error(`Error initializing logger: ${e.message}`);
+            // without logger multiple things will have undefined behavior and probably more is wrong -> do not start
+            throw new Error(`Error initializing logger: ${e.message}`);
```

Comment:
```diff
- e.message
+ e.stack
```
to see where the error is coming from.

Also, does throwing here do a clean shutdown or should we just log and `process.exit` instead?

Reply:
Makes sense for these "strange" errors, I assume it makes no difference but we can call process exit